### DISCUSSION TITLE
When CodeViz finishes running but the popped-up page shows nothing.

### DISCUSSION THREAD

**Discussion by 9alaxy** [vizzuhq/codeviz#21] [Q&A]
1. First, i use  popup menu of the folders in explorer
![image](https://github.com/vizzuhq/codeviz/assets/98274374/212519ad-2a39-4234-8c71-d9a940a19a32)
2. And then "CodeCounter" work .
![image](https://github.com/vizzuhq/codeviz/assets/98274374/f8cdd66c-ee72-4f48-8f8e-da0a6e592a05)
3. Finally，CodeViz page popped up ，but there is nothing in it.
![image](https://github.com/vizzuhq/codeviz/assets/98274374/5dac27c6-e7e9-460c-8080-71892f329931)

**Comment by jazzystu** [comment]
Also having the same issue with remote dev environment configuration. VC Code Counter is working

**Reply by jazzystu** [reply]
Seeing multiple 401 errors in dev tools:
![Screenshot 2024-05-30 124324](https://github.com/vizzuhq/codeviz/assets/10567321/591cbaa9-46cf-4004-b25d-00a91c79535c)
